scripts/maintenance/virsh-cleanup

```diff
@@ -1,5 +1,12 @@
 #!/bin/sh
 
+echo "Warning: This will destroy effectively all libvirt resources"
+read -p "Continue? " -n 1 -r
+if [[ ! $REPLY =~ ^[Yy]$ ]]
```

Comment:
Can I just change it to `/bin/bash` instead?

Reply:
> Can I just change it to `/bin/bash` instead?

The Control-C/enter approach seems usable enough to me that I don't think it's worth breaking POSIX compat.  But most folks will have Bash, so I don't feel too strongly about it.  And I'm not an approver anyway ;).

Reply:
An alternative to require an explicit Y/y in POSIX would be somerthing like

```sh
printf 'Warning: This will destroy effectively all libvirt resources\nContinue [yN]? '
read CONTINUE
if test "${CONTINUE}" != y -a "${CONTINUE}" != Y
then
	echo 'Aborted' >&2
	exit 1
fi
```

Reply:
@cgwalters I don't want to break POSIX compatibility since _I_ depend on that to use these scripts. If it has to be bash, you need to use `#!/usr/bin/env bash`, but I like @wking's first suggestion better.

Reply:
OK, done with @wking 's suggestion :arrow_down: